OpenRA.Mods.Common/Widgets/Logic/Editor/MapToolsLogic.cs

```diff
@@ -0,0 +1,82 @@
+#region Copyright & License Information
```

Comment:
it would definitely be better to generate the sprites ingame rather than use art assets. Something like `FillRect` from `RgbaColorRenderer` should do the trick

Reply:
Will this work for isometric tiles?

Reply:
you specify the corners of the quad, so yes